cost-model/src/cost_tracker.rs

```diff
@@ -886,38 +885,6 @@ mod tests {
                     assert_eq!(expected_block_cost, *units);
                 });
         }
-
-        // adjust overflow
```

Comment:
tests for over/underflow removed. atomic doesn't do this for fetch_add/sub...unless there's a bug elsewhere we shouldn't ever be adjusting by such large numbers. If we were adjusting up/down by u64::MAX it'd already be creating silent issues by pegging to u64::MAX (can't pack any more) or to 0 (can pack forever if repeated)